site/src/pages/CreateWorkspacePage/CreateWorkspacePage.tsx

```diff
@@ -1,111 +1,103 @@
 import { useMachine } from "@xstate/react"
```

Comment:
The `mode=auto` seems to be work except for required parameters (without `default` property):

<img width="1209" alt="Screenshot 2023-07-26 at 11 14 09" src="https://github.com/coder/coder/assets/14044910/67d3d698-e1a0-4a87-a6bf-3600ed4e4c6c">

Reply:
This is a good one, thx for catching it.